Readme.md

```diff
@@ -1,2 +1,99 @@
 # XAF Blazor - How to Implement a TreeList Editor
 
+To add the TreeList Editor to your ASP.NET Core Blazor application, implement the following components:
```

Comment:
`Implement the following components to add the TreeList Editor to your ASP.NET Core Blazor application:`